package.json

```diff
@@ -1,35 +1,29 @@
 {
-  "name": "local-home-virual-environment",
+  "name": "local-home-testing",
   "version": "0.0.1",
-  "description": "",
+  "description": "Testing environment for Local Fulfillment Apps. Mocks behavior of Local Home Platform.",
+  "author": "Google LLC",
   "private": true,
+  "main": "./build/src/index.js",
   "license": "Apache-2.0",
   "scripts": {
     "compile": "tsc",
-    "test": "ava --verbose",
-    "lint": "eslint ."
+    "lint": "tslint --project .",
```

Comment:
shouldn't we be using gts?

Reply:
I hadn't realized `tslint` was deprecated / not compatible with gts.  I'll switch back to `eslint`. Opened #33